benchmarks/llm/perf.sh

```
@@ -14,42 +14,103 @@
+if [ -z "${concurrency_array+x}" ]; then
+    concurrency_array=($concurrency_list)
+fi
+
+# Validate concurrency values
+validate_concurrency
+
+echo "Running genai-perf with:"
+echo "Model: $model"
+echo "ISL: $isl"
+echo "OSL: $osl"
+echo "Concurrency levels: ${concurrency_array[@]}"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)